src/lib/http-post-parser.ts

```diff
@@ -4,9 +4,9 @@ import {
   WebRequestOnBeforeRequestEventDetails,
   // WebRequestOnBeforeSendHeadersEventDetails,
 } from "../types/browser-web-request-event-details";
-import { escapeString } from "./string-utils";
+// import { escapeString } from "./string-utils";
```

Comment:
Observation: Seems like some stuff is getting commented out as a way to save it for later.

Reply:
Yes indeed. The post parsing code is currently kept in place, but commented, in case it is to be used in the near future.

Reply:
See https://github.com/mozilla/openwpm-webext-instrumentation/issues/18